tests/yast2_gui/yast2_control_center.pm

```diff
@@ -40,9 +40,6 @@ sub search {
         send_key 'backspace';
     }
     wait_screen_change { type_string $name; } if $name;
```

Comment:
IMHO even the `wait_screen_change` is wasteful here as we do something like `assert_and_click` waiting for the right screen afterwards anyway.

Reply:
So you mean we can speed up this test even more? :)

Reply:
yes we can. But then someone else adds back `wait_still_screen` and `wait_screen_change` because they are too scared to add an `assert_screen` instead or do not understand how os-autoinst works so we should probably go slow on the optimization :)